live-examples/html-examples/input/input.html

```diff
@@ -0,0 +1,24 @@
+<fieldset>
+    <legend>Input</legend>
+    <form>
+        <label for="display-name"> Name:
+            <span class="warning">*(Allows only alphabets.)</span> 
+        </label>
+        <input type="text" name="display-name" 
```

Comment:
This element is missing an `id` attribute with a value that matches the `for` attribute of the associated `label` element.